scripts/release/shared-commands/parse-params.js

```diff
@@ -38,12 +38,8 @@ module.exports = async () => {
   const params = commandLineArgs(paramDefinitions);
 
   if (params.build !== null) {
-    if (params.commit !== null) {
-      console.error(
-        '`build` and `commmit` params are mutually exclusive. Choose one or the other.`'
-      );
-      process.exit(1);
-    }
+    // TODO: Should we just remove the `build` param? Seems like `commit` is a
+    // sufficient replacement.
```

Comment:
I'll do this.

Reply:
Looks like the [release script README](https://github.com/facebook/react/blob/master/scripts/release/README.md) is in a broken state at the moment. I'll take a pass at that too with an update PR.